task-launcher/src/tasks/shared/trials/finishExperiment.js

```diff
@@ -1,12 +1,29 @@
 import { jsPsych } from "../../taskSetup";
-import { taskStore } from "../helpers";
+import { taskStore, PageAudioHandler } from "../helpers";
+import { mediaAssets} from '../../..'; 
 
 
 export function finishExperiment() {
     const t = taskStore().translations;
     jsPsych.endExperiment(
-      `<div id="prompt-container-text">
-                <p id="prompt">${t?.taskCompletion || 'Thank you for completing the task! You can now close the window.'}</p>
-            </div>`,
-    ); // ToDo: style text and add audio message?
+        `<div class='instructions-container'>
```

Comment:
Can you replace this with `lev-stimulus-container`? We want to get rid of all `task.scss` declarations.

Reply:
Sure! I forgot to check for that on this file, thanks for the reminder